src/common/exif.cc

```diff
@@ -659,6 +659,10 @@ static bool dt_exif_read_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
         img->exif_focus_distance = (FocusDistanceLower + FocusDistanceUpper) / 200;
```

Comment:
This code (that i wrote) is incorrect though.
It should be
```c++
    else if(Exiv2::testVersion(0,25,0) && FIND_EXIF_TAG("Exif.CanonFi.FocusDistanceUpper"))
    {
      float FocusDistanceUpper = pos->toFloat();
      if(FocusDistanceUpper <= 0.0f || FocusDistanceUpper >= 0xffff)
      {
        img->exif_focus_distance = 0.0f;
      }
      else
      {
        img->exif_focus_distance = FocusDistanceUpper / 100.0;
        if(FIND_EXIF_TAG("Exif.CanonFi.FocusDistanceLower"))
        {
          float FocusDistanceLower = pos->toFloat();
          if(FocusDistanceLower > 0.0f && FocusDistanceLower < 0xffff))
          {
            img->exif_focus_distance += FocusDistanceLower / 100.0;
          }
        }
      }
    }
```

Reply:
Ok, I was indeed asking myself about the / 200 ! I'll integrate your change in this PR. Thanks for the review.

Reply:
Actually, wait, this is almost correct.
We still need to average, so 
```c++
img->exif_focus_distance += FocusDistanceLower / 100.0;
img->exif_focus_distance /= 2.0;
```